### PR TITLE
ChangeStream Examples

### PR DESCRIPTION
**Pull request by jlord**
This branch contains examples for using the new ChangeStream feature which will be used elsewhere in MongoDB in documentation 📖

### CODE REVIEW

File: test/functional/operation_changestream_example_tests.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+              const newChangeStream = collection.watch({ resumeAfter: resumeToken });
+
+              newChangeStream.next(function(err, next) {
+                if (err) return console.log(err);
+                expect(err).to.equal(null);
+                expect(next).to.exist;
+                client.close();
+                done();
+              });
+              // End Changestream Example 3
```
Comment (mbroadst) [inline test/functional/operation_changestream_example_tests.js]: it looks like the end of the example is omitting a few levels of braces here? Looks like you should just move this "end changestream" tag three lines lower

File: test/functional/operation_changestream_example_tests.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
+'use strict';
+var setupDatabase = require('./shared').setupDatabase;
+var expect = require('chai').expect;
+
+// ./node_modules/.bin/mongodb-test-runner -l -e replicaset test/functional/operation_changestream_example_tests.js
```
Comment (mbroadst) [inline test/functional/operation_changestream_example_tests.js]: let's not commit this please

File: test/functional/operation_changestream_example_tests.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
+'use strict';
+var setupDatabase = require('./shared').setupDatabase;
```
Comment (mbroadst) [inline test/functional/operation_changestream_example_tests.js]: might as well make these `const`

File: test/functional/operation_changestream_example_tests.js (hunk 2 above)
Comment (jlord) [reply test/functional/operation_changestream_example_tests.js]: 😢 👋

**Pull request merged**